web/src/components/navbar/NavBar.test.tsx

```diff
@@ -204,6 +204,21 @@ describe("<NavBar />", () => {
         expect(screen.queryByText(Config.navigationDefaults.profileLinkText)).not.toBeInTheDocument();
         expect(screen.queryByText(Config.navigationDefaults.registerButton)).not.toBeInTheDocument();
       });
+
+      it("does not render nav menu for an onboarding authenticated user", async () => {
+        setLargeScreen(false);
+        render(
```

Comment:
pebble - i'd look into why your fix didn't result in failing test. We may need to create some.

Reply:
discussed: Not an issue.